core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala

```diff
@@ -49,16 +49,23 @@ class SnapshotTableColumnNames(config: Config) {
   override def toString: String = s"SnapshotTableColumnNames($persistenceId,$sequenceNumber,$created,$snapshot)"
 }
 
-class SnapshotTableConfiguration(config: Config) {
+class SnapshotTableConfig(config: Config) {
```

Comment:
No need to rename old configuration classes. I just suggested to keep the name for the new one consistent with the existing pattern.

Reply:
Why do not keep consistency across all classes?

Reply:
Let's do it in a separate PR. As long as two devs work on the same part of the codebase, refactoring should be postponed.